src/Evolution/Protocols.hpp

```diff
@@ -0,0 +1,33 @@
+// Distributed under the MIT License.
```

Comment:
Until the discussion with whether or not to continue to have protocols in the (evolution) code is settled, please stop introducing them.